doc/docs.md

```diff
@@ -1993,20 +1993,13 @@ mut p := Point{
 	x: 10
 	y: 20
 }
-p = Point{
-	x: 30
-	y: 4
-}
+
+// The field name can be omitted.
+p = Point{30, 4}
+
 assert p.y == 4
-//
-// array: first element defines type of array
-points := [Point{10, 20}, Point{20, 30}, Point{40, 50}]
-println(points) // [Point{x: 10, y: 20}, Point{x: 20, y: 30}, Point{x: 40,y: 50}]
 ```
 
-Omitting the struct name also works for returning a struct literal or passing one
-as a function argument.
-
```

Comment:
Do not delete these.

Reply:
? wait, you said only these 2 lines are needed..
```
// The field name can be omitted.
p = Point{30, 4}`
```